test/task.h

```diff
@@ -215,4 +215,15 @@ UNUSED static int can_ipv6(void) {
   return supported;
 }
 
+UNUSED static const char* localhost_ipv4() {
+  #ifdef __FreeBSD__
```

Comment:
Nits: can you write `localhost_ipv4(void)` instead of `localhost_ipv4()` and not indent the `#ifdef` statement (and dedent the code inside)?

Can you add a comment explaining why this function exists?